src/server/redis.js

```diff
@@ -6,7 +6,7 @@ import redisMock from 'redis-mock'
 const client =
   process.env.NODE_ENV === 'test'
     ? redisMock.createClient()
-    : redis.createClient()
+    : redis.createClient(process.env.REDIS_HOST || '127.0.0.1')
```

Comment:
Are you sure we don’t need to pass any port?

Reply:
We could if we plan on using a non-standard port (createClient uses default 6379)

Reply:
Okay cool. Thought you’d have to pass the port.